specification/cognitiveservices/OpenAI.Inference/models/completions/extensions_onyourdata_azure_cognitive_search_options.tsp

```diff
@@ -60,7 +66,11 @@ enum AzureCognitiveSearchQueryType {
 A specific representation of configurable options for Azure Cognitive Search when using it as an Azure OpenAI chat
 extension.
 """)
-model AzureCognitiveSearchChatExtensionConfiguration {
+@added(ServiceApiVersions.v2023_08_01_Preview)
+model AzureCognitiveSearchChatExtensionConfiguration
+  extends AzureChatExtensionConfiguration {
+  ...OnYourDataCommonChatExtensionConfiguration;
```

Comment:
I might be misreading this, but I think all these properties go under a "parameters" property?

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/AzureOpenAI/inference/preview/2023-12-01-preview/inference.json#L1945

Reply:
You're completely correct. This representation was wrong -- the merge of the branch @jpalvarezl has (#26790) addresses this inconsistency.